src/components/MaterialUI/muiThemes.js

```diff
@@ -473,33 +578,84 @@ const setThemeOverrides = theme => ({
 	MuiTableContainer: {},
 	MuiTablePagination: {},
 	MuiTableSortLabel: {},
-	MuiTableFooter: {},
+	MuiTableFooter: {
+		...theme.MuiTableFooter,
+		root: {
+			...theme.root,
+			backgroundColor: theme.palette.grey.lighter,
+		}
+	},
+	MuiInput: {
+		...theme.MuiInput,
+		underline: {
+			border: "none",
+			"&:before" : {
+				content: "none"
+			},
+		}
+	},
 	MuiInputBase: {
 		...theme.MuiInputBase,
+		root: {
+			...theme.root,
+			"& + .MuiButton-root, & + .MuiIconButton-root, & + .MuiInputBase-root" : {
+				marginLeft: theme.spacing(1)
+			}
+		},
 		input: {
 			...theme.input,
-			borderRadius: 4,
+			borderRadius: theme.shape.borderRadius,
 			position: "relative",
 			backgroundColor: theme.palette.background.paper,
 			border: `1px solid ${theme.palette.grey.borders}`,
 			fontSize: theme.typography.fontSize,
 			maxWidth: theme.spacing(20),
+			minWidth: theme.spacing(14),
 			padding: theme.spacing(0.6, 0.6, 0.6, 0.6),
 			transition: theme.transitions.create(["border-color", "box-shadow"]),
+			"&:focus, &:active": {
+				borderRadius: theme.shape.borderRadius,
+				borderColor: theme.palette.focus,
+				boxShadow: `0 0 4px ${theme.palette.focus}`,
+				outline: "none",
+			},
+		},
+		adornedStart : {
+			"& > .MuiOutlinedInput-root:first-child, & > .MuiButtonBase-root:first-child": {
+				borderTopRightRadius: 0,
+				borderBottomRightRadius: 0,
+			},
+		},
+		adornedEnd : {
+
+		},
+		inputAdornedStart: {
+			...theme.inputAdornedStart,
+			borderRight: "none",
+			borderTopLeftRadius: 0,
+			borderBottomLeftRadius:0,
+
+		},
+		inputAdornedEnd: {
+			...theme.inputAdornedEnd,
+			borderLeft: "none",
+			borderTopRightRadius: 0,
+			borderBottomRightRadius:0,
+			"& + .MuiOutlinedInput-root, & + .MuiButtonBase-root": {
+				borderTopLeftRadius: 0,
+				borderBottomLeftRadius:0,
+			},
 		},
 	},
 	MuiSelect: {
 		...theme.MuiSelect,
 		select: {
 			...theme.select,
 			minWidth: theme.spacing(15),
-			borderRadius: 4,
-			"&:focus": {
-				borderRadius: 4,
-				borderColor: "#4fa1f0",
-				boxShadow: "0 0 4px #4fa1f0",
-				outline: "none",
```

Comment:
@a-rousseau, you did not want to keep the focus styling, I thought it would be better to keep it, especially about the border radius being 0 when having the focus.

Reply:
@joclefrancois the styling was defined in a more global components like MuiInputBase-input, MuiButtonBase-root. so more components will benefit from this